.vscode/launch.json

```diff
@@ -20,7 +20,7 @@
       "name": "Debug Tests",
       "type": "Ruby",
       "request": "launch",
-      "program": "${workspaceRoot}/${input:ecosystem}/.bundle/bin/rspec",
+      "program": "${workspaceRoot}/omnibus/.bundle/bin/rspec",
```

Comment:
What was the rationale for this? I don't see it mentioned in the commit messages / PR description, so was surprised to see it... did I just miss it somehow?

Reply:
This change was necessitated by https://github.com/dependabot/dependabot-core/pull/5529